web-client/src/ducks/auth/facebook/actions.ts

```diff
@@ -0,0 +1,33 @@
+import {
+  facebookLoginWithFirebasePopUp,
+  getRedirectResult,
+  loginWithFirebaseRedirect,
+} from 'src/http/resources/auth';
```

Comment:
I was wondering, these functions are the operations that are to be carried out for an `action creator` and since these functions are again created as modules (because all the functions related to auth are put in auth.ts in resources directory they wouldn't really be used again for an `action creator`), wouldn't it be better for us to define these in the same directory as the action creators in a file called functions.ts? 
I also think that putting all auth related functions in `src/http/resources/auth` is valid only for the old manner of redux modules now we have sub redux modules, so I think these should also be having hierarchy)

Reply:
I split some of the http functions into different files. Although we should think of a better way fo splitting this.

Reply:
This was corrected